Heavy.js

```diff
@@ -0,0 +1,23 @@
+const Soldier = require('./Soldier');
+
+function Heavy() {
+    Soldier.apply(this, arguments);
+    this.damageResist = 0.2;
```

Comment:
it was not in the specification, but since it not used outside it should be private